framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -82,7 +82,6 @@ qx.Mixin.define("qx.ui.core.MExecutable",
      */
     _bindableProperties :
     [
-      "enabled",
```

Comment:
Nevertheless, it is not fixable. The Command overrides the enabled state - which is bad, because it shouldn't have the lead there. It should only treat widgets as enabled, if they are supposed to be enabled.

Even if you attach the "execute" to the Command, it doesn't honour the states. So I'd just drop the "Command can enable or disable Button *states*" behaviour. I personally don't expect a Command to disable Widgets. I'd expect a disabled Command not to react on anything. That's the case.